src/lib/Components/Inbox/Conversations/Previews/PDFPreview.tsx

```diff
@@ -56,5 +63,6 @@ export default Relay.createContainer(PDFPreview, {
 interface RelayProps {
   pdfAttachment: {
     file_name?: string
+    download_url?: string
```

Comment:
Oh, my bad, I didn’t really mean for you to change these typings in this TS source (that will get overwritten on a next relay2ts run anyways), I actually meant that the MP schema should probably have `GraphQLNonNull`?